tests/spec/date_series_ops/test_date_comparisons.py

```diff
@@ -129,3 +131,26 @@ def test_is_equal_with_str_date(spec_test):
             4: None,
         },
     )
+
+
+def test_is_in_dates(spec_test):
+    dates = [date(2010, 1, 1), date(2010, 1, 2)]
```

Comment:
I might be tempted to inline these. These tests get formatted as part of the documentation and so it's good to keep syntactic clutter to a minimum. Obviously for a longer list of dates pulling them out into a variable like this would help readability. But as it's only two dates I think it's clearer without the extra line.